Classes/Steno.sc

```diff
@@ -216,10 +216,14 @@ Steno {
 			if(names.isNil or: { names.includes(token) }) {
 				args = argList.at(i);
 				oldSynth = oldSynthList.at(i);
-				newSynth = this.newSynth(token, nil, args, oldSynth.nodeID);
+
+				server.openBundle;
```

Comment:
why is this needed? In the method `eval`, a bundle is opened anyhow already. I tend to use the outermost context for organizing bundling, just to keep the code simple.

Reply:
removed bundling